forge-gui/res/cardsfolder/c/crackling_doom.txt

```diff
@@ -5,5 +5,5 @@ A:SP$ DealDamage | Cost$ R B W | Defined$ Player.Opponent | NumDmg$ 2 | SubAbili
 SVar:DBRepeat:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBSac
 SVar:DBChooseCard:DB$ ChooseCard | Defined$ Player.IsRemembered | Choices$ Creature.greatestPowerControlledByRemembered | ChoiceTitle$ Choose a creature you control with the greatest power | Mandatory$ True | RememberChosen$ True
 SVar:DBSac:DB$ SacrificeAll | ValidCards$ Card.IsRemembered | SubAbility$ DBCleanup | StackDescription$ Each opponent sacrifices a creature with the greatest power among creatures they control.
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearChosen$ True
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```

Comment:
I think we'll still need ClearChosen for the last time through the loop

It would be nice to add a toggle to ChooseCardEffect to allow using addChosenCards() (which would also need to be added) instead of always needing to use setChosenCards()

Reply:
ClearChosen doesn't exist
ClearChosenCard isn't needed because it' not copied when moving zones anyway